Makefile

```diff
@@ -86,6 +86,23 @@ testbuild:
 	  $(GO) test $(GOFLAGS) -tags '$(TAGS)' -o "$$DIR"/"$$OUT" -ldflags '$(LDFLAGS)' "$$p" $(TESTFLAGS) || exit 1; \
 	done
 
+# Build all tests into DIR and strips each.
+# DIR is required.
+.PHONY: testbuildall
+testbuildall: TESTS := $(shell $(GO) list $(PKG))
+testbuildall: GOFLAGS += -c
+testbuildall:
+ifndef DIR
```

Comment:
alternative heckling against `DIR`:

`find . -type f -name '*.test' -exec mv '{}' "${DIR}{}" ';'` in the script will do the same thing.